EpiResponsivePicture/ResizedImage/ResizedImageFormat.cs

```diff
@@ -0,0 +1,9 @@
+namespace Forte.EpiResponsivePicture.ResizedImage
+{
+    public enum ResizedImageFormat
+    {
+        Jpg,
```

Comment:
What about having default enum value of "PreserveOriginal", instead of having null value? Default value is more meaningful than null so we shoul avoid nulls when possible